src/Compilers/CSharp/Test/Emit/PrivateProtected.cs

```diff
@@ -67,7 +72,7 @@ void M()
     }
 }
 ";
-            CreateStandardCompilation(source, parseOptions: TestOptions.Regular7_2)
+            var compilation = CreateStandardCompilation(source, parseOptions: TestOptions.Regular7_2)
```

Comment:
This change seems incomplete. Did you intend to use `compilation`?

Reply:
I was going to add a semicolon and make the next line compilation.VerifyDiagnostics(), but I got distracted while making the change.

---
In reply to: [137602381](https://github.com/dotnet/roslyn/pull/21742#discussion_r137602381) [](ancestors = 137602381)